test/CMakeLists.txt

```diff
@@ -78,19 +78,27 @@ if(UMF_BUILD_LIBUMF_POOL_DISJOINT)
                  LIBS disjoint_pool)
 endif()
 
-if(UMF_BUILD_LIBUMF_POOL_JEMALLOC)
-    add_umf_test(NAME jemalloc_pool
-                 SRCS pools/jemalloc_pool.cpp malloc_compliance_tests.cpp
-                 LIBS jemalloc_pool)
+if(UMF_BUILD_OS_MEMORY_PROVIDER)
+    if(UMF_BUILD_LIBUMF_POOL_JEMALLOC)
+        add_umf_test(NAME jemalloc_pool
+                    SRCS pools/jemalloc_pool.cpp malloc_compliance_tests.cpp
+                    LIBS jemalloc_pool)
+    endif()
+else()
+    message(STATUS "The jemalloc_pool test is DISABLED, because UMF_BUILD_OS_MEMORY_PROVIDER is set to OFF")
```

Comment:
jemalloc could use any memory provider to work - I would change this message to "The jemalloc_pool test is DISABLED, because it uses OS_MEMORY_PROVIDER, while UMF_BUILD_OS_MEMORY_PROVIDER flag is set to OFF" or similar

Reply:
Done